shared/domain/src/main/java/gov/va/vro/model/mas/MasAutomatedClaimPayload.java

```diff
@@ -90,7 +90,7 @@ public Boolean isPresumptive() {
   }
 
   @JsonIgnore
-  public Integer getClaimId() {
+  public Integer getBenefitClaimId() {
     return claimDetail == null ? null : Integer.parseInt(claimDetail.getBenefitClaimId());
   }
```

Comment:
I don't think the types match. `vbmsId` (aka BenefitClaimId) is a String. We shouldn't be coercing it into an Integer. Any uses of this method is likely to produce a bug.

Reply:
Corrected, and good catch.